### PR TITLE
Add poster frames to main videos - PMT #111496

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: worthapp/index.html [hunk 1]
```diff
@@ -1291,7 +1291,7 @@ <h1 class="display-3">
   Beginning the Journey
  </h3>
  <div class="embed-responsive embed-responsive-16by9">
-  <video class="embed-responsive-item" controls="" poster="img/lucia.png">
+  <video class="embed-responsive-item" controls="" poster="img/.jpg">
```
Comment (nbuonin) [inline worthapp/index.html]: Should this be 'carol.jpg'?

File: worthapp/index.html [hunk 2]
```diff
@@ -1327,7 +1327,7 @@ <h1 class="display-3">
  <h3>
  </h3>
  <div class="embed-responsive embed-responsive-16by9">
-  <video class="embed-responsive-item" controls="" poster="img/lucia.png">
+  <video class="embed-responsive-item" controls="" poster="img/.jpg">
```
Comment (nbuonin) [inline worthapp/index.html]: Also, 'carol.jpg' here?

File: worthapp/index.html (hunk 1 above)
Comment (nikolas) [inline worthapp/index.html]: this video doesn't have a defined poster image yet - i guess i should leave it blank when that's the case but my templating system isn't that advanced yet

Comment (nbuonin) [inline worthapp/index.html]: ah ok, cool

Comment (nikolas) [inline worthapp/index.html]: Yeah, it's not obvious but what's going on here is `worthapp/index.html` is really just an output file that I'm committing, that's generated with the `make build` command, which pulls together all the templates in `scripts/templates/`

**Pull request merged**